### PR TITLE
minor testcases added

### PR DESCRIPTION
**Pull request by robonetphy**
minor test cases added

### CODE REVIEW

File: test/testcases.md [hunk 1]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
 
 - [ ] `defaultBlock` property
+  - [ ] If `string` passed
+    - [ ] If passed `string` in tools,the passed tool should be used as default tool.
+    - [ ] If passed `string` not in tools,the Paragraph tool should be used as default tool.
+  - [ ] If omitted the Paragraph tool should be used as default tool.
+
+- [ ] `sanitizer` property
+  - [ ] If `object` passed
+    -[ ] The Editor.js should be clean the HTML tag according to mentioned configuration.
```
Comment (neSpecc) [inline test/testcases.md:52]:
```suggestion
    - [ ] The Editor.js should clean the HTML tags according to mentioned configuration.
```

File: test/testcases.md [hunk 2]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
+  - [ ] If `VERBOSE` passed,the editor should outputs all messages to the console.
+  - [ ] If `INFO` passed,the editor should outputs info and debug messages to the console.
+  - [ ] If `WARN` passed,the editor should outputs only warn messages to the console.
+  - [ ] If `ERROR` passed,the editor should outputs only error messages to the console.
+  - [ ] If omitted the editor should outputs all messages to the console.
 
 - [ ] `defaultBlock` property
+  - [ ] If `string` passed
+    - [ ] If passed `string` in tools,the passed tool should be used as default tool.
+    - [ ] If passed `string` not in tools,the Paragraph tool should be used as default tool.
```
Comment (neSpecc) [inline test/testcases.md:47]:
```suggestion
    - [ ] If passed `string` not in the `tools` option, the Paragraph tool should be used as the default tool.
```

File: test/testcases.md [hunk 3]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
-- [ ] `tools` property
+  - [ ] If `VERBOSE` passed,the editor should outputs all messages to the console.
+  - [ ] If `INFO` passed,the editor should outputs info and debug messages to the console.
+  - [ ] If `WARN` passed,the editor should outputs only warn messages to the console.
+  - [ ] If `ERROR` passed,the editor should outputs only error messages to the console.
+  - [ ] If omitted the editor should outputs all messages to the console.
 
 - [ ] `defaultBlock` property
+  - [ ] If `string` passed
+    - [ ] If passed `string` in tools,the passed tool should be used as default tool.
```
Comment (neSpecc) [inline test/testcases.md:46]:
```suggestion
    - [ ] If passed `string` in the `tools` option, the passed tool should be used as the default tool.
```

File: test/testcases.md [hunk 4]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
+    - [ ] If passed `string` not in tools,the Paragraph tool should be used as default tool.
+  - [ ] If omitted the Paragraph tool should be used as default tool.
+
+- [ ] `sanitizer` property
+  - [ ] If `object` passed
+    -[ ] The Editor.js should be clean the HTML tag according to mentioned configuration.
+  - [ ] If omitted the Editor.js should be initialize with default `sanitizer` configuration, which allow the tags like `paragraph`, `anchor` and `bold` for cleanning HTML.
```
Comment (neSpecc) [inline test/testcases.md:53]:
```suggestion
  - [ ] If omitted the Editor.js should be initialized with the default `sanitizer` configuration, which allows the tags like `paragraph`, `anchor`, and `bold` for cleaning HTML.
```

File: test/testcases.md [hunk 5]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
+  
+- [ ] `tools` property
+  - [ ] If `object` passed
+    -[ ] Editor.js should be initialized with the Passed tools only.
```
Comment (neSpecc) [inline test/testcases.md:57]:
```suggestion
    - [ ] Editor.js should be initialized with the Passed tools only.
```

File: test/testcases.md [hunk 6]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
+  - [ ] If omitted Editor.js should be initialized with the Paragraph tool only.
```
Comment (neSpecc) [inline test/testcases.md:58]:
```suggestion
  - [ ] If omitted 
    - [ ] Editor.js should be initialized with the Paragraph tool only.
```

File: test/testcases.md [hunk 7]
```diff
@@ -7,33 +7,52 @@ This document will describe various test cases of the editor.js functionality. F
...
+    - [ ] If passed `string` not in tools,the Paragraph tool should be used as default tool.
+  - [ ] If omitted the Paragraph tool should be used as default tool.
+
+- [ ] `sanitizer` property
+  - [ ] If `object` passed
+    -[ ] The Editor.js should be clean the HTML tag according to mentioned configuration.
+  - [ ] If omitted the Editor.js should be initialize with default `sanitizer` configuration, which allow the tags like `paragraph`, `anchor` and `bold` for cleanning HTML.
+  
+- [ ] `tools` property
+  - [ ] If `object` passed
```
Comment (neSpecc) [inline test/testcases.md:56]: I think, we also need to check the structure of the `tools` available options

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: test/testcases.md (hunk 7 above)
Comment (robonetphy) [reply test/testcases.md:56]: @neSpecc can you describe it with some examaple.

**Comment by robonetphy** [comment]
@neSpecc,
Should I include test cases for
`readOnly`, `i18n`, `onReady`, `onChange` and `data` ?

**Comment by neSpecc** [comment]
> @neSpecc,
> Should I include test cases for
> `readOnly`, `i18n`, `onReady`, `onChange` and `data` ?

yes, but you can do it via separate PR.

**Comment by khaydarov** [comment]
@ranemihir is this PR ready for review?

**Comment by robonetphy** [comment]
Yup it is

**Review by neSpecc — APPROVED** [review]

**Pull request merged**